contracts/drafts/dao/DAO.sol

```diff
@@ -0,0 +1,117 @@
+pragma solidity ^0.5.10;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+import "../../issuance/IssuanceEth.sol";
+import "../token/ERC20MultiDividendable.sol";
+
+
+contract DAO is ERC20Mintable, ERC20MultiDividendable, IssuanceEth {
+
+    mapping(address => uint256) public begAmount;
+    uint256 totalAmounts;
+    mapping(address =>
+        mapping(address => uint256)
+    ) public votesForIdeaByHolder;
+    mapping(address => uint256) public totalVotesForIdea;
+    mapping(address => uint256) public totalVotesByHolder;
+    mapping(address => address[]) public backersForIdea;
+    event Here();
+
+    constructor()
+    ERC20Mintable()
+    ERC20MultiDividendable()
+    IssuanceEth(address(this))
+    public
+    {
+        _createState("NEVER");
+        _createTransition("LIVE", "SETUP");
+        _createTransition("FAILED", "SETUP");
+        addMinter(address(this));
+    }
+
+    function transferFunds(address payable _wallet) public onlyOwner {
+        require(currentState == "NEVER", "You can never do this.");
+    }
+
+    function restartFundingRound() public onlyOwner {
+        require(
+            currentState == "LIVE" || currentState == "FAILED",
+            "Initial funding round not ended."
+        );
+        _transition("SETUP");
+    }
+
+    function begMoneyForIdea(uint256 amount, address payable idea) public {
+        require(currentState == "LIVE", "Founders not defined yet.");
+        require(
+            totalAmounts + amount <= address(this).balance,
+            "You beg too much."
+        );
+        begAmount[idea] = amount;
+    }
+
+    function voteForIdea(uint256 votes, address idea) public {
+        require(currentState == "LIVE", "Founders not defined yet.");
```

Comment:
I think the error message is a bit confusing. You can just say "Idea needs to be LIVE."

Reply:
No, I could not. It's not about the current state of the idea, it's about the DAO. But, somehow, youa re right, I think I could just say `"DAO needs to be LIVE"`.